src/views/guidelines/guidelines.jsx

```diff
@@ -0,0 +1,51 @@
+var React = require('react');
+var FormattedMessage = require('react-intl').FormattedMessage;
+
+var render = require('../../lib/render.jsx');
+var Page = require('../../components/page/page.jsx');
+var Box = require('../../components/box/box.jsx');
+
+require('./guidelines.scss');
+
+var Guidelines = React.createClass({
+    type: 'Guidelines',
+    render: function () {
+        return (
+            <Box title={<FormattedMessage id='guidelines.title' />}>
```

Comment:
The entire return should be wrapped inside of a 
```(html)
<div className="inner">
    <Box>
    .
    .
    .
    </Box>
</div>
```
That should handle centering the box on the page for ya.